filesharer.py

```diff
@@ -0,0 +1,12 @@
+from filestack import Client
+
+
+class FileSharer:
+    def __init__(self, filepath, api_key="AW6dnnV2cS2Gl0vhvqZKWz"):
+        self.filepath = filepath
+        self.api_key = api_key
+
+    def share(self):
+        client = Client(self.api_key)
```

Comment:
I haven't investigated how filestack works. Is it possible to move this line into `_ _init_ _` section?